schemas/dictionary.json

```diff
@@ -78,6 +78,7 @@
       "title": "Path",
       "description": "A fully qualified URL, or a POSIX file path..",
       "type": "string",
+      "pattern": "^[^./~]",
```

Comment:
Could you please add a few tests for it? 

The specs have a `test` folder and I think it could be simple to test compiled JSON Schema using `tv4` against some paths.

Reply:
@roll done, please see the commit below